package.json

```diff
@@ -52,7 +53,31 @@
         }
       ]
     },
+    "customEditors": [
+      {
+        "viewType": "cfg.editor",
+        "displayName": "Cfg Editor",
+        "selector": [
+          {
+            "filenamePattern": "*.cfg"
+          }
+        ],
+        "priority": "default"
+      }
+    ],
     "commands": [
+      {
+        "command": "onevscode.flip-cfg-to-ui",
+        "title": "Open cfg (UI)",
+        "category": "ONE",
+        "icon": "$(go-to-file)"
+      },
+      {
+        "command": "onevscode.flip-cfg-to-ini",
+        "title": "Open cfg (ini)",
```

Comment:
I guess that you wanted to mention _text mode_ editing here.
Then something like `Open in text editor` would be better I think because
- cfg and ini are file format
- cfg and ini file format can be edited in graphical editor or text editor.